### DISCUSSION TITLE
Video url not working

### DISCUSSION THREAD

**Discussion by abdulhadi240** [deepgram/community#499] [General help] [answered]
from deepgram import DeepgramClient , PrerecordedOptions

deepgram = DeepgramClient()

source = {
    "url": "https://www.youtube.com/watch?v=RuO5SHVtSZ8"
}

# STEP 2 Call the transcribe_url method on the prerecorded class
options = PrerecordedOptions(
    model="nova",
    smart_format=True,
    summarize="v2",
)
url_response = deepgram.listen.prerecorded.v("1").transcribe_url(source, options)
transcript = url_response.results.channels[0].alternatives[0].transcript
durations = url_response.metadata.duration

print(transcript , durations )

Every thing works fine with audio url , but i want to use video . plz help!!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
You can't directly link to a YouTube page since it's a webpage and not that actual video stream (mp4).

If you do a simple curl GET:
```
curl https://www.youtube.com/watch?v=RuO5SHVtSZ8
```

You should notice that you are getting HTML returned to you.
